lib/resources/r4/appointment.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: Appointment
+field_name_base_url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment
+fields:
+- name: resourceType
```

Comment:
since there's no binding: key here, resourceType will not be rendered to the terminology bindings table (which is fine)

this will be needed for the create documentation but for now we can take out resourceType

Reply:
removed https://github.com/cerner/fhir.cerner.com/pull/269/commits/6e094421299a9712e721c6d4a8d83ba8a4e36b87#diff-fbb215a73c336ca1ac34d91de2fb411eL5